sei-tendermint/types/node_info_test.go

```diff
@@ -178,76 +180,74 @@ func TestNodeInfoAddChannel(t *testing.T) {
 	require.Contains(t, nodeInfo.Channels, byte(0x02))
 }
 
+func TestResolveAddressStringDNS(t *testing.T) {
+	addr, err := ResolveAddressString("deadbeefdeadbeefdeadbeefdeadbeefdeadbeef@localhost:1234")
+	require.NoError(t, err)
+	require.True(t, addr.Addr().IsLoopback())
+	require.Equal(t, 1234, addr.Port())
+}
```

Comment:
I am curious, why this test stands separate from the rest? Could we add this case to the table-test below?